packages/dr.rb

```diff
@@ -0,0 +1,19 @@
+require 'package'
+
+class Dr < Package
+  description 'A simple ed-like devRant client'
+  homepage 'https://github.com/Ewpratten/dr'
+  version '2.0'
+  source_url 'https://github.com/Ewpratten/dr/archive/2.0-cros.tar.gz'
+  source_sha256 '4e39a28f947ac044e28349fb10ffb423100fb00acb0ab819e931d931e3f0b02f'
```

Comment:
Maybe this will help clear up the checksum hash confusion?
```
$ wget https://github.com/Ewpratten/dr/archive/2.0-cros.tar.gz
$ sha256sum 2.0-cros.tar.gz 
0ab9003c23d74d1ca8b0cfaf33e7db5e5b908d3448c87c4c0a83c9bf77a8ac23  2.0-cros.tar.gz
```
```
- source_sha256 '4e39a28f947ac044e28349fb10ffb423100fb00acb0ab819e931d931e3f0b02f'
+ source_sha256 '0ab9003c23d74d1ca8b0cfaf33e7db5e5b908d3448c87c4c0a83c9bf77a8ac23'
```